src/lib/gql.ts

```diff
@@ -41,7 +41,7 @@ const createInMemoryCache = () =>
 
 type Config = {
 	gateway: string;
-	jwt: string;
+	jwt: string | undefined;
```

Comment:
I'm unclear why this can now be `undefined`?
The type is not exported (not used anywhere else) and there's no other changes to this file or the ApolloProvider which is a parent of AuthProvider

Reply:
that's likely because of this?
https://github.com/icgc-argo/rdpc-ui/pull/114#discussion_r1404368017

Reply:
Justin is correct, due to updating `egoJwt` definition, createApploClient args (in `ApolloProvider`) are invalid without this change